src/main/java/List.java

```diff
@@ -31,6 +31,24 @@ public void printAddedTask() {
         Omoh.printHorizontalLine();
     }
 
+    public void addTaskAndDeadline (String input) {
+        Deadline extractedInfo = Deadline.extractTaskAndDueDate(input);
+        tasks[totalTasks] = new Deadline(extractedInfo.description, extractedInfo.by);
+        totalTasks++;
+        printTaskAndDeadline(extractedInfo);
+    }
+
+    public void printTaskAndDeadline(Deadline description) {
+        Omoh.printHorizontalLine();
+        System.out.print("     ");
+        System.out.println("Got it. I've added this task:");
+        System.out.print("       ");
+        System.out.println("[D][ ]" + description.description + " (by: " + description.by + ")" );
```

Comment:
The [D] part could be added as a boolean in the method so it could be easier to adjust in the future